corehq/ex-submodules/couchforms/analytics.py

```diff
@@ -97,6 +97,14 @@ def get_last_form_submission_received(domain):
     return submission_time
 
 
+def get_300th_form_submission_received(domain):
```

Comment:
Seems like a random place to put this method. Why not https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/calculations.py? I think this is where most of the calculations like this live